### PR TITLE
Training info

### LINKED ISSUE #256: Correct training info
A lot of what's currently stated on the website for training is not true e.g.

> "We offer a range of training courses in topics such as Programming, High Performance Computing and Software Engineering. Most of these are free to members of the University of Sheffield."

The content should be checked and corrected.

### PR DESCRIPTION
**Pull request by bobturneruk**
Resolves #256

### CODE REVIEW

**Comment by bobturneruk** [comment]
This must be reviewed by @willfurnass or @mondus as it's a significant change to team-university communications.

File: _data/navigation.yml [hunk 1]
```diff
@@ -16,12 +16,8 @@
   subitems:
     - title: Overview
       url: /training/
-    - title: Training Events
-      url: /training/events
```
Comment (willfurnass) [inline _data/navigation.yml:22]: Seems reasonable to remove given we're not doing much training at present and we have a separate events page.

File: _data/navigation.yml [hunk 2]
```diff
@@ -16,12 +16,8 @@
...
     - title: Code Clinic
       url: /support/code-clinic
-    - title: Software and Data Carpentry
-      url: /training/carpentry
```
Comment (willfurnass) [inline _data/navigation.yml:26]: Yes, this page is quite out of date now (list of instructors, Carpentries membership) so it makes sense to remove it, at least until we've got the resources to run Carpentries workshops semi-regularly again.

However, I think we should incorporate the first para into the main 'Support and Training' page to explain what Carpentries workshops are.

File: pages/community/index.md [hunk 3]
```diff
@@ -7,9 +7,6 @@ type: text
 
 The RSE group supports research software developers (any researcher who develops software) and research software engineers
 across the university through our community activities.
-We run regular social events and seminars
-as well as providing training code help.
-Many of these activities are completely free.
```
Comment (willfurnass) [inline pages/community/index.md:12]: We might want to explicitly list Lunch Bytes here but this can be a separate issue.

**Comment by bobturneruk** [comment]
Have you finished reviewing @willfurnass ?

**Comment by bobturneruk** [comment]
@willfurnass @mondus - just giving this a bump! I'd like to get this merged before the newsletter goes out so if we start driving traffic here, the information is up to date.

File: pages/training/index.md [hunk 4]
```diff
@@ -1,13 +1,25 @@
...
 
-In the future we aim to run a DDP module on Software Engineering best practice. The design of this module is currently under review for delivery in 2019.
\ No newline at end of file
+Some example topics in which we have expertise include:
+
+- Python
+- R
+- Matlab
+- Reproducible Research
+- C, C+, C++
```
Comment (willfurnass) [inline pages/training/index.md]: C+ is not a thing.

File: pages/training/index.md [hunk 5]
```diff
@@ -1,13 +1,25 @@
...
-Check out out complete list of [training events](./events) or find out more about [Software and Data Carpentry](./carpentry).
+The RSE team can deliver of a range of training courses. Several members of the team are ["Software Carpentry"](https://software-carpentry.org/) trainers and we can tailor and deliver these courses. We have recently delivered training commissioned by Doctoral Training Centres on "Reproducible Research Data & Project Management in R" and on use of the Linux shell for accessing HPC systems.
 
-In the future we aim to run a DDP module on Software Engineering best practice. The design of this module is currently under review for delivery in 2019.
\ No newline at end of file
+Some example topics in which we have expertise include:
+
+- Python
+- R
+- Matlab
```
Comment (willfurnass) [inline pages/training/index.md]: MATLAB all caps

File: pages/training/index.md [hunk 6]
```diff
@@ -1,13 +1,25 @@
...
+- Reproducible Research
+- C, C+, C++
+- git and GitHub
+- Linux shell
+- High Performance Computing
+- GPU Computing
```
Comment (willfurnass) [inline pages/training/index.md:38]: Add Docker and Ansible

**Comment by bobturneruk** [comment]
from @mondus via slack:
>Having a quick skim I don't think there is anything to controversial. Might want to mention Twins DL training and plans for training support around Toer 2 facilities which have RSE support (JADE, Bede)

File: pages/training/index.md [hunk 7]
```diff
@@ -1,13 +1,25 @@
...
 ---
 
-We offer and support the delivery of a range of training courses in topics such as Programming, High Performance Computing and Software Engineering. Most of these are free to members of the University of Sheffield.
+We offer a limited "free at the point of use" support service: [code clinics](../support/code-clinic.md) which allow researchers to spend time with RSEs working on problems. More general software support within the University of Sheffield is provided by [IT Services](https://www.sheffield.ac.uk/it-services/research/support), who also offer a range of [free training courses](https://www.sheffield.ac.uk/it-services/research/training/timetable).
 
-Check out out complete list of [training events](./events) or find out more about [Software and Data Carpentry](./carpentry).
+The RSE team can deliver of a range of training courses. Several members of the team are ["Software Carpentry"](https://software-carpentry.org/) trainers and we can tailor and deliver these courses. We have recently delivered training commissioned by Doctoral Training Centres on "Reproducible Research Data & Project Management in R" and on use of the Linux shell for accessing HPC systems.
```
Comment (willfurnass) [inline pages/training/index.md]: s/Software Carpentry trainers/certified Carpentries instructors/

**Comment by bobturneruk** [comment]
I will rebase.

File: pages/training/index.md [hunk 8]
```diff
@@ -6,10 +6,20 @@ type: text
 permalink: /training/
 ---
 
-We offer and support the delivery of a range of training courses in topics such as Programming, High Performance Computing and Software Engineering. Most of these are free to members of the University of Sheffield.
+We offer a limited "free at the point of use" support service: [code clinics](../support/code-clinic.md) which allow researchers to spend time with RSEs working on problems. More general software support within the University of Sheffield is provided by [IT Services](https://www.sheffield.ac.uk/it-services/research/support), who also offer a range of [free training courses](https://www.sheffield.ac.uk/it-services/research/training/timetable).
```
Comment (willfurnass) [inline pages/training/index.md]: "limited" - overly negative?

File: pages/training/events/index.md [hunk 9]
```diff
@@ -1,13 +0,0 @@
...
-title: Training Events
-layout: page
-permalink: /training/events/
----
-
-We offer a range of training courses in topics such as Programming, High Performance Computing and Software Engineering. Most of these are free to members of the University of Sheffield.
-
-Here are details of current and previous training events we've run. If you'd like us to rerun any of them, please get in touch and we'll see what we can do.
-
-If you would like **email notifications about future training events**, please join our [mailing-list](/community/).
```
Comment (willfurnass) [inline pages/training/events/index.md:11]: Presumably don't need to promote this Google Group now switching to using more formal newsletter for comms (#263).

File: pages/training/index.md (hunk 4 above)
Comment (willfurnass) [reply pages/training/index.md]: Fixed in 59cc58c2187edb256248be36353a8ed90eb65e1a

File: pages/training/index.md (hunk 5 above)
Comment (willfurnass) [reply pages/training/index.md]: Fixed in 59cc58c2187edb256248be36353a8ed90eb65e1a

File: pages/training/index.md (hunk 7 above)
Comment (willfurnass) [reply pages/training/index.md]: Fixed in 59cc58c2187edb256248be36353a8ed90eb65e1a

File: pages/training/index.md (hunk 8 above)
Comment (bobturneruk) [reply pages/training/index.md]: Yes - should be removed.

**Review by willfurnass — APPROVED** [review]

**Comment by bobturneruk** [comment]
Thanks @willfurnass !

**Comment by willfurnass** [comment]
Thank you @bobturneruk for identifying some of the stale parts of the site!

**Pull request merged**